examples-bkup/chat.rs

```diff
@@ -172,3 +207,64 @@ fn print_ourinfo(crust: &mut Crust) {
         serde_json::to_string(&ourinfo).unwrap()
     );
 }
+
+fn parse_cli_args() -> CliArgs {
+    let matches = App::new("Simple chat app built on Crust")
+        .about(
+            "This chat app connects two machines directly without intermediate servers and allows \
+             to exchange messages securely. All the messages are end to end encrypted.",
+        )
+        .arg(
+            Arg::with_name("listening_port")
+                .help(
+                    "Optional server Crust will be listening for incoming connections ON. If \
```

Comment:
Same thing about Crust. And it probably could use a bit of a rephrasing:
```suggestion
                    "Optional port number which the peer will use to listen for incoming connections. If \
```